### PR TITLE
Profile Pictures

### PR DESCRIPTION
**Pull request by mmontori1**
Fixes # 227

### PR Description
Created a component which handles uploading and displaying a user's profile picture. All that it needs is passing a users avatars by props. I changed up the FileUpload component a bit so you can pass the type of file it's uploading by props so you can use it for both resumes and profile pictures.

### What needs to be reviewed?
Mostly look through the changes to the uploadProfile action as well as the ProfilePicture component. I'm currently displaying the user's profile picture in the profile.jsx page, we can remove it if it's not needed there. Also the way I'm handling if a profile picture has been uploaded is by checking if the avatars' length is greater than 2, which works but idk if its the best way ¯\_(ツ)_/¯

### CODE REVIEW

File: app/pages/home/sponsorship.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
     Brick
 } from './section_components.jsx';
 
-import SponsorLogos from '../../components/SponsorLogos.jsx';
+import SponsorLogos from '../../components';
```
Comment (AndrewBoland) [inline app/pages/home/sponsorship.jsx]: This breaks the home page currently, change it to 

Import { SponsorLogos } from ‘../../components’;

Or just revert it

File: app/components/ProfilePicture.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Avatar = styled.img`
+    width: inherit;
```
Comment (AndrewBoland) [inline app/components/ProfilePicture.jsx]: We should probably set maxWidth here so the image isn’t displayed too huge. Same with min maybe

**Comment by AndrewBoland** [comment]
It looks good to me, just gotta merge with master and fix prettier.

**Comment by mmontori1** [comment]
I think the master branch needs to be fixed with prettier?

**Comment by connorkrupp** [comment]
@mmontori1 I promise after this merge conflict is fixed then we "good af"

**Pull request merged**